### ISSUE TITLE
caffe2 spec/SRPM

### ISSUE THREAD

**Issue by advorkin** [facebookarchive/caffe2#951]
Hello,

I've created caffe2 RPM file compatible with RHEL7.  Please let me know if you are interested in spec/SRPM files.

Thank you.

**Comment by leemgs** [comment]
@advorkin, Did you distribute RPM version of Caffe2 for Linux distro such as RHEL7/CentOS7?

**Comment by myungjoo** [comment]
@leemgs Why don't you send PR with your RPM spec file? I know you already have one. :) I think ```caffe2.spec.in``` with some macro should be fine.

**Comment by leemgs** [comment]
@myungjoo , Yes. I am going to submit a PR to support the RPM spec file soon.

**Comment by advorkin** [comment]
@leemgs That's correct.  Please let me know if you'd like it.  Thank you.

**Comment by hjoon0510** [comment]
@advorkin Where can we download a RPM file  (e.g., caffe2.rh7.rpm) that you generated with a spec file for RHEL7+Caffe2? Could you share an web address of the RPM file for the Caffe2 users that use RHEL7?

**Comment by myungjoo** [comment]
@advorkin Would you please send pull-request to this repo or share your RPM spec in your fork?
